extensions/amp-base-carousel/0.1/test-e2e/test-multi-visible-single-advance.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {getNextArrow, getPrevArrow, getSlides, sleep} from './helpers';
+
+describes.endtoend(
+  'AMP carousel advancing with multiple visible',
+  {
+    testUrl:
+      'http://localhost:8000/test/fixtures/e2e/amp-carousel/0.1/multi-visible-single-advance.amp.html',
```

Comment:
This file should probably be in `amp-base-carousel/` not `amp-carousel/`

Reply:
If moved there, you should also be able to remove all the duplicated img assets.

Reply:
Fair point--although, I do think that in general, `fixtures/e2e` is the right place to put these test files (in the future).

Reply:
Yeah, this shoudl have a master issue as well if possible. Would be a huge code health win IMO